### PR TITLE
Copy MVP confirmation page and logic

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
This copies over the logic and content from the MVP minus the wording for "increased" conditions. I also removed the 4142 content since we'll be integrating that form right into the middle of this one.

## Testing done
Manually tested and copied the unit tests.

### Scenario 1: Successfully submitted to EVSS
image
Because I couldn't get a successful submission yet, I had to mock this out. Act like there's a confirmation number where it says there should be. The code is there, I promise.

### Scenario 2: Check back later
image

### Scenario 3: Non-retryable error
image

## Acceptance criteria
- [x] All three scenarios show up under the right circumstances

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/constants.js [hunk 1]
```diff
@@ -141,3 +141,21 @@ export const PTSD = 'ptsd';
 export const NINE_ELEVEN = '2001-09-11';
 
 export const ERR_MSG_CSS_CLASS = '.usa-input-error-message';
+
+export const submissionStatuses = {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/constants.js]: ARE WE NOT GOING WITH CAPS LOCK FOR THESE?

File: src/applications/disability-benefits/all-claims/constants.js [hunk 2]
```diff
@@ -141,3 +141,21 @@ export const PTSD = 'ptsd';
...
+  // Statuses returned by the API
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/constants.js]: Does this duplicate the constants in Increase Only? Should we just point those refs to the All Claims constants file and remove those old ones?

File: src/applications/disability-benefits/all-claims/constants.js [hunk 3]
```diff
@@ -141,3 +141,21 @@ export const PTSD = 'ptsd';
...
+  pending: 'submitted', // Submitted to EVSS, waiting response
+  retry: 'retrying',
+  succeeded: 'received', // Submitted to EVSS, received response
+  exhausted: 'exhausted', // EVSS is down or something; ran out of retries
+  failed: 'non_retryable_error', // EVSS responded with some error
+  // When the api serves a failure
+  apiFailure: 'apiFailure',
+};
+
+export const terminalStatuses = new Set([
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/constants.js]: same question here

**Review by markgreenburg — APPROVED** [review]
LGTM. I wish we could reduce the duplication of the polling wrapper and `<ConfirmationPage/>` but I'm not sure there's a way to do this without unnecessary work, since each relies on the specific versions of its children that it imports from either `all-claims/` or `526EZ/`

File: src/applications/disability-benefits/all-claims/constants.js (hunk 1 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/constants.js]: Hehehe, I thought about it. I just didn't really care for `SUBMISSION_STATUSES.PENDING` or whatever. :man_shrugging:

I'm alright with changing them. I just don't like the way it looks.

**Pull request merged**